Fixes #28678 - Modify hammer cli azurem README (#7)
theforeman/hammer_cli_foreman_azure_rm: README.md

## Description
CLI tool for AzureRM compute resource in Foreman [foreman_azure_rm](https://github.com/theforeman/foreman_azure_rm)

## Installation

### Package
```
# yum install tfm-rubygem-hammer_cli_foreman_azure_rm
```

### Foreman Installer
```
# foreman-installer --enable-foreman-cli-azure
# hammer -r
```

> Please note that using **hammer -r** will load command options for azurerm compute resource for the first time.

Reviewer: ```suggestion
Please note that **hammer -r** will reload Apipie cache so that command options for azurerm compute resource will get loaded.
```
